src/imputation/tmi_imputation.py

```diff
@@ -464,16 +472,16 @@ def run_tmi(
     TMILogger.info("Imputing for R&D type (civil or defence).")
     df = impute_civil_defence(df)
 
-    TMILogger.info("Calculating the trimmed mean for target variables")
-    df = tmi_pre_processing(df, target_variables)
+    lf_target_variables = config["imputation"]["lf_target_vars"]
```

Comment:
I notice this is inconsistent with the shortform target variable import `sf_target_variables = list(config["breakdowns"])`

Can we change one or the other to make them consistent?

Reply:
I agree, I'll change one of them.